### ISSUE TITLE
Bio.Phylo.write generate invalid newick format output

### ISSUE THREAD

**Issue by wpwupingwp** [biopython/biopython#1315]
`from Bio import Phylo as p;
a=p.parse('20.nwk','newick');
p.write(a, '20.nwk.biopython','newick')`

`diff 20.nwk 20.nwk.biopython`
1c1
< (T0:0.0693147181,T12:0.1309333320,((T16:0.0366725280,T18:0.3296837366)99:0.0701179352,((T11:0.0314710745,T6:0.0752897185)86:0.1052683357,(T2:0.0738144546,T7:0.4135166557)40:0.0605136303,(T8:0.0556869562,(T10:0.0044997366,T19:0.0293029679)39:0.0339677368)100:0.0245900538,(T1:0.0454130280,T14:0.0052346480)95:0.0368169323,(T9:0.0230671818,T5:0.6214608098)93:0.1537117251,(T3:0.0860383100,(T13:0.2796881415,T17:0.0055512710)99:0.0012)82:0.0353821875,(T4:0.0807436327,T15:0.1406497068)99:0.0512493681)100:0.1703748592)60:0.1382302340);

\> (T0:0.06931,T12:0.13093,((T16:0.03667,T18:0.32968):99.00:0.07012,((T11:0.03147,T6:0.07529):86.00:0.10527,(T2:0.07381,T7:0.41352):40.00:0.06051,(T8:0.05569,(T10:0.00450,T19:0.02930):39.00:0.03397):100.00:0.02459,(T1:0.04541,T14:0.00523):95.00:0.03682,(T9:0.02307,T5:0.62146):93.00:0.15371,(T3:0.08604,(T13:0.27969,T17:0.00555):99.00:0.00120):82.00:0.03538,(T4:0.08074,T15:0.14065):99.00:0.05125):100.00:0.17037):60.00:0.13823):0.00000;

The output places **extra colon** (":") behind bootstrap value, which causes the output cannot be parsed by other software even biopython itself.

Seems [this issus(#888)](https://github.com/biopython/biopython/issues/888) is related to this problem.

Ping Wu

**Comment by wpwupingwp** [comment]
Update:
Same problem found in nexus format output.

**Comment by peterjc** [comment]
Which version of Biopython are you using?

CC @etal

**Comment by wpwupingwp** [comment]
Biopython 1.69

**Comment by jvollme** [comment]
I also have this problem (Biopython 1.70). Newick and Nexus trees generated by Bio.Phylo are not readable at all by any tool. Also in the PhyloXML format, bootstraps are not recognized (by ITOL at least).
Is there some kind of workaround for this?

**Comment by Caiofcas** [comment]
Was trying to reproduce but apparently it's fixed.

Versions:
```
3.9.5 (default, May 14 2021, 00:00:00) 
[GCC 10.3.1 20210422 (Red Hat 10.3.1-1)]
CPython
Linux-5.12.9-200.fc33.x86_64-x86_64-with-glibc2.32
1.79
```
Code:

```python
>>> import sys
>>> import io
>>> from Bio import Phylo as p
>>> tree_str = "(T0:0.0693147181,T12:0.1309333320,((T16:0.0366725280,T18:0.3296837366)99:0.0701179352,((T11:0.0314710745,T6:0.0752897185)86:0.1052683357,(T2:0.0738144546,T7:0.4135166557)40:0.0605136303,(T8:0.0556869562,(T10:0.0044997366,T19:0.0293029679)39:0.0339677368)100:0.0245900538,(T1:0.0454130280,T14:0.0052346480)95:0.0368169323,(T9:0.0230671818,T5:0.6214608098)93:0.1537117251,(T3:0.0860383100,(T13:0.2796881415,T17:0.0055512710)99:0.0012)82:0.0353821875,(T4:0.0807436327,T15:0.1406497068)99:0.0512493681)100:0.1703748592)60:0.1382302340);"
>>> t = p.parse(io.StringIO(tree_str),"newick")
>>> p.write(t, sys.stdout, "newick")
(T0:0.06931,T12:0.13093,((T16:0.03667,T18:0.32968)99.00:0.07012,((T11:0.03147,T6:0.07529)86.00:0.10527,(T2:0.07381,T7:0.41352)40.00:0.06051,(T8:0.05569,(T10:0.00450,T19:0.02930)39.00:0.03397)100.00:0.02459,(T1:0.04541,T14:0.00523)95.00:0.03682,(T9:0.02307,T5:0.62146)93.00:0.15371,(T3:0.08604,(T13:0.27969,T17:0.00555)99.00:0.00120)82.00:0.03538,(T4:0.08074,T15:0.14065)99.00:0.05125)100.00:0.17037)60.00:0.13823):0.00000;
```

It no longer seems to add the incorrect semicolons, unless I am missing something.

**Comment by peterjc** [comment]
This was fixed in a0352e16d5208cbdab76e1fffb7c334ea1e34c74 from #1276, included in Biopython 1.71 onwards.

Belatedly closing issue, thank you all.
